stack-8.4.4.yaml

```diff
@@ -1,3 +0,0 @@
-resolver: lts-12.26
```

Comment:
Decided to only keep the 3 most recent minor versions of GHC, so removing this older one.